### ISSUE TITLE
Needs maintainers

### ISSUE THREAD

**Issue by thommay** [berkshelf/vagrant-berkshelf#321]
Chef does not (and hasn't for a long time) recommend using this plugin - we think test-kitchen is a better pattern for all the work we do.
Given that, we need to find maintainers that use this plugin who are happy to review code, fix issues and make releases.
@timvisher @Tcharl @tknerr y'all have contributed work to this plugin, would you be interested in helping to maintain it?

**Comment by Tcharl** [comment]
Hi @thommay ,

I'm really a ruby noob, and I don't even know how to compile and use it ^^. So sorry, maybe when I'll be more confortable with all that stuff. I use chef and vagrant as a hobby on my spare time so...

However, I'm now using chef, but having both is kinda nice ^^.
Regards

**Comment by timvisher** [comment]
Sadly I can't help. I agree that test-kitchen is a fundamentally better option.

**Comment by thommay** [comment]
OK. Status right now is that in the absence of new maintainers we'll likely do a release to support vagrant 2.0 and then mark this as entirely deprecated.

**Comment by timvisher** [comment]
That sounds very reasonable to me.

**Comment by tknerr** [comment]
Sorry, same here. Not using this plugin anymore... 

+1 for a vagrant 2.0 compatibility release though, in case someone still uses it.

**Comment by etki** [comment]
Actually i've been using this plugin simply to provision my dev box (it's not a test box, i just need to pull other cookbooks to run my own, e.g. use `elasticsearch` cookbook to bring up ES). As far as i've understood this is not considered as a common use case scenario, so i feel now like i'm missing something and the whole world has been doing this in other way. Can you shed some light on how provision dependencies should be installed?

**Comment by Tcharl** [comment]
Yes, as a devbox, this plugin is very nice :-). Before testing, we should sometimes be able to instantiate components ^^.

**Comment by timvisher** [comment]
@etki When a few bugs hit me a while ago I realized that I could do what this plugin was doing for me by writing a very basic script that first ran `berks vendor` into the cookbook directory of the project and then run `vagrant X`. That's what I would recommend you do. Obviously add the vendored cookbooks directory to your gitignore or whatever.

**Comment by etki** [comment]
@timvisher that's a pity, i thought there was another fully-automated way (building some project that is erm windows guy-oriented and the less the mess, the better). Thank you!

**Comment by timvisher** [comment]
@etki Yeah, it is a bit of a shame. FWIW, that's basically what this plugin is so if you wanted to take over as maintainer you'd have your solution. :)

**Comment by gavvvr** [comment]
It is not clear to me what should I use if my goal is to set up a virtual machine with Vagrant and apply cookbooks (without testing).
Which Workflow is supposed now? `kitchen converge` with supplying settings in Vagranrfile like this?
```
driver:
  name: vagrant
  vagrantfiles:
    - VagrantfileKitchen.rb
```

**Comment by timvisher** [comment]
@gavvvr You set up your `Vagrantfile` to reference a directory managed by `berks vendor`.

I pushed up a quick little demo of this. I script most of it. https://github.com/timvisher/berks-demo

**Comment by timvisher** [comment]
@gavvvr Under no circumstances should you use `kitchen` to managed your actual dev boxes. `kitchen` is purely for testing. It can provision a box and you could use it for that but you're really hitting a screw with a hammer at that point.

**Comment by etki** [comment]
@timvisher i'm terribly sorry, i would love to, but i guess we share the same not-enough-time case :/ I know that OSS doesn't live by itself, but i know in advance i won't handle it :\

**Comment by gavvvr** [comment]
@timvisher great, thank you! Dependencies are resolved even without `berks vendor`. I just use `vagrant up` and it does all the job.

**Comment by timvisher** [comment]
@gavvvr That sounds a lot like you still have `vagrant-berkshelf` installed. What's the output of `vagrant plugin list` look like for you?

**Comment by gavvvr** [comment]
@timvisher You are right. I tried on a clean Vagrant and it complained on missing `berks-demo/berks-cookbook`. I've read that it is better to use Policyfile for new Chef cookbooks. I wonder if it is possible to resolve Policyfile dependencies and apply recipes with single `vagrant up` command (like I did with `vagrant-berkshelf` installed). Could not find any example with Vagrantfile+Policyfile

**Comment by timvisher** [comment]
@gavvvr I hadn't heard from anyone that they're moving away from Berksfiles but maybe so. No matter what I think dependency resolution is not really considered a vagrant problem at the moment hence this plugin and any like it getting orphaned. I'd expect any Policyfile solution to be in the same boat (a plugin could be written and maintained to automate the resolution but it won't generally be maintained by core vagrant devs).
